### DISCUSSION TITLE
突然无法访问了，请求大佬帮助 x509

### DISCUSSION THREAD

**Discussion by LitGrass** [v2fly/v2ray-core#1486] [Q&A] [answered]
昨天晚上还能上，今早起来就不行了，报错信息如下
`v2ray.com/core/proxy/vmess/outbound: failed to find an available destination > v2ray.com/core/common/retry: [v2ray.com/core/transport/internet/websocket: failed to dial WebSocket > v2ray.com/core/transport/internet/websocket: failed to dial to (wss://www.XXXXXX.COM/1e/):  > x509: certificate has expired or is not yet valid] > v2ray.com/core/common/retry: all retry attempts failed `

配置信息如下：
端口：443
传输协议：ws
伪装类型：none
底层传输安全：tls

现状：
IP良好

**Comment by kingwilliam** [comment, marked as answer]
"certificate has expired or is not yet valid"
“证书已过期或尚未生效”

看看您的证书

**Reply by LitGrass** [reply]
重装了一下，可以了。感谢回复！

**Reply by yl-emon-coder** [reply]
重装了服务端？
